apps/framework-cli/src/cli/routines/scripts.rs

```diff
@@ -261,6 +263,87 @@ pub async fn terminate_workflow(
     }))
 }
 
+pub async fn terminate_all_workflows(project: &Project) -> Result<RoutineSuccess, RoutineFailure> {
+    let client_manager = Arc::new(TemporalClientManager::new(&project.temporal_config));
+
+    let request = ListWorkflowExecutionsRequest {
+        namespace: DEFAULT_TEMPORTAL_NAMESPACE.to_string(),
+        page_size: 1000,
+        query: "ExecutionStatus = 'Running'".to_string(),
+        ..Default::default()
+    };
+
+    let response = client_manager
+        .execute(|mut client| async move {
+            client
+                .list_workflow_executions(request)
```

Comment:
There is no pagination on this? all is returned?

Reply:
Their api has a `page_size` which I'm setting as 1000. Should that be smaller?

Reply:
I think that's ok for now - maybe add a comment in there.

Reply:
Okay cool, updated